src/main/kotlin/com/pubnub/api/subscribe/Subscribe.kt

```diff
@@ -6,15 +6,19 @@ import com.pubnub.api.PubNub
 import com.pubnub.api.PubNubError
 import com.pubnub.api.PubNubException
 import com.pubnub.api.eventengine.EffectDispatcher
-import com.pubnub.api.eventengine.EventEngineConf
-import com.pubnub.api.eventengine.Sink
-import com.pubnub.api.managers.EventEngineManager
+import com.pubnub.api.eventengine.Event
+import com.pubnub.api.eventengine.EventEnginesConf
 import com.pubnub.api.managers.ListenerManager
+import com.pubnub.api.managers.PresenceEventEngineManager
+import com.pubnub.api.managers.SubscribeEventEngineManager
+import com.pubnub.api.presence.eventengine.PresenceEventEngine
+import com.pubnub.api.presence.eventengine.effect.PresenceEffectFactory
+import com.pubnub.api.presence.eventengine.effect.effectprovider.HeartbeatProviderImpl
+import com.pubnub.api.presence.eventengine.effect.effectprovider.LeaveProviderImpl
+import com.pubnub.api.presence.eventengine.event.PresenceEvent
```

Comment:
I don't think the package `c.p.a.subscribe` should depend on `c.p.a.presence`. It make sense to use just subscribe without presence. `Subscribe` class should be concerned only with subscribe and presence related things should go into `c.p.a.presence.Presence`